src/main/java/seedu/foodiebot/logic/commands/SelectItemCommand.java

```diff
@@ -2,19 +2,106 @@
 
 import static java.util.Objects.requireNonNull;
 
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.List;
+import java.util.Optional;
+import java.util.logging.Logger;
+
+import seedu.foodiebot.commons.core.LogsCenter;
+import seedu.foodiebot.commons.core.index.Index;
+import seedu.foodiebot.commons.util.JsonUtil;
 import seedu.foodiebot.model.Model;
+import seedu.foodiebot.model.ReadOnlyFoodieBot;
+import seedu.foodiebot.model.UserPrefs;
+import seedu.foodiebot.model.budget.Budget;
+import seedu.foodiebot.model.food.Food;
+import seedu.foodiebot.storage.JsonAdaptedBudget;
 
 /** Select the current list view item. */
 public class SelectItemCommand extends Command {
     public static final String COMMAND_WORD = "select";
 
-    public static final String MESSAGE_USAGE = "";
+    public static final String MESSAGE_USAGE =
+            COMMAND_WORD
+            + "Parameters: "
+            + "FOOD_INDEX \n"
+            + "Example: "
+            + COMMAND_WORD
+            + " "
+            + "1 ";;
+    public static final String MESSAGE_SUCCESS = "You have selected: %s, this costs: $%.2f\n";
+    public static final String MESSAGE_SUCCESS_BUDGET = "You have selected: %s, this costs: $%.2f\n"
```

Comment:
select item uses the message from budget?

Reply:
Oh I meant, if a budget is set, it uses that success message.
If no budget is set, then it uses the regular message success